services/buy.py

```diff
@@ -1,81 +1,77 @@
 import datetime
 import math
-
 from sqlalchemy import select, update, func
-
 import config
-from db import async_session_maker
+from db import session_execute, session_commit, session_refresh, get_db_session
 from models.buy import Buy
 from models.user import User
 from services.user import UserService
 from utils.other_sql import RefundBuyDTO
 
 
 class BuyService:
-    buys_per_page = config.PAGE_ENTRIES
 
     @staticmethod
     async def get_buys_by_buyer_id(buyer_id: int, page: int):
-        async with async_session_maker() as session:
-            stmt = select(Buy).where(Buy.buyer_id == buyer_id).limit(BuyService.buys_per_page).offset(
-                page * BuyService.buys_per_page)
-            buys = await session.execute(stmt)
+        async with get_db_session() as session:
```

Comment:
maybe we should rename "session" to "db_session"; at least I'm getting a warning in my IDE

Reply:
@6R1M5L07H, In my IDE the warning also appears, this is a Pycharm bug, even if we rename the session it won't help, because the reason for this bug is in @asynccontextmanager